packages/web3-validator/src/errors.ts

```diff
@@ -54,7 +54,11 @@ export class Web3ValidatorError extends Error {
 			this.errors.length
 		} error[s]:\n${this._compileErrors().join('\n')}`;
 
-		Error.captureStackTrace(this, Web3ValidatorError);
+		if (typeof Error.captureStackTrace === 'function') {
+			Error.captureStackTrace(this, Web3ValidatorError);
+		} else {
+			this.stack = new Error().stack;
+		}
```

Comment:
Why not do this in `packages/web3-errors/src/web3_error_base.ts` so that all errors receive the functionality?

Reply:
Good question! Actually, this class in `web3-validator` takes care of an array of validation errors. And it does not have `code` number like the one in `web3-errors/src/web3_error_base.ts`. So, the classes in `web3-errors` and  `web3-validator` are different. However, there could be a different task to refactor their code if this is desired.

Reply:
There should be same web3error base class used consistently in all packages, so could you make changes in web3 validator for using web3error base class from web3-errors package, and create error codes. ( Either use Web3Error base class here or move these error classes in Error package + use  Web3Error base class )

Reply:
Ok